Sources/MarathonCore/Command.swift

```diff
@@ -90,6 +93,8 @@ extension Command {
             return ""
         case .help:
             return ""
+        case .export:
+            return "<script-path> [<export-path>] [--force]"
```

Comment:
@JohnSundell Do we enforce the order of arguments and flags? For example what should the expected behavior be for the following commands:

`marathon export --force myScript`

`marathon export myScript --force ./newDir`